src/app/services/ServicesInfoPage.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import Header from '../../components/header';
+import Accordion from '../../components/accordion';
+import Selector from '../../components/selector';
+
+class ServicesInfoPage extends Component {
+  state = { active: -1 };
+
+  onToggleOpen = value =>
+    this.setState(({ active }) => ({ active: active !== value ? value : -1 }));
+
+  render() {
+    const { active } = this.state;
+    const { services } = this.props;
+
+    return (
+      <div className="text-left">
+        <div className="py-5 px-3 container">
+          <Header>What programs and services are available at this location?</Header>
+        </div>
+        <Accordion>
```

Comment:
The accordions aren't iterating on the data but simply renders any child content. Maybe I'm misunderstanding the question.

Reply:
Ha, right. I meant not the accordion components themselves, but rather what's being passed to them here, in ServicesInfoPage. :)
At least at the time I was making the comment, it was hard-coded rather than being based on the category data.